src/app/solicitation/summary/results-detail/results-detail.component.html

```diff
@@ -20,8 +20,19 @@
             <div style="padding-bottom: 5px"><a target="_blank" href="{{solicitation.url}}">{{solicitation.url}}</a></div>
             <div *ngIf="solicitation.reviewRec !== 'Not Applicable' " style="padding-bottom: 5px">
               <span style="font-weight:600">Results:&nbsp;</span>
-              <span style="font-weight:500" >{{solicitation.reviewRec}}</span>
-
+              <span style="font-weight:500" >{{solicitation.reviewRec}}</span>                    
+            <!-- insert button here for linking to ART -->
+            </div>
+            <div class="row">
+              <div class="col-md-8">
+                <a href="https://section508.gov/art" target="_blank">
```

Comment:
I'll be addressing the vulnerability in the bootstrap 5 update, but for this can you add ```rel="noopener noreferrer"``` into the <a> tag class. This mitigates the tabnabbing security concerns.

Reply:
added